lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -542,8 +542,8 @@ spec = do
             ]
 
         ["key", "root", "--help"] `shouldShowUsage`
-            [ "Usage:  key root [--wallet-style WALLET_STYLE] MNEMONIC_WORD..."
-            , "                 [--encoding KEY-ENCODING]"
+            [ "Usage:  key root [--wallet-style WALLET_STYLE]"
+            , "                 [--encoding KEY-ENCODING] MNEMONIC_WORD..."
```

Comment:
This just makes me realize: the mnemonic words should also probably be read from stdin instead of passed as argument!